src/css/profile/mobile/common/badge.less

```diff
@@ -0,0 +1,71 @@
+.ui-badge {
+    &::after {
+        content: attr(title);
+        .font(medium);
+        font-size: 11 * @sp_base;
+        text-align: center;
+        border-radius: 8.5 * @px_base;
+        position: absolute;
+        background-color: var(--badge-color);
+        top: -7 * @px_base;
+        right: -4 * @px_base;
+        padding: 3 * @px_base 3 * @px_base;
+        color: var(--color-white);
+        min-width: 11 * @px_base; // 17 - 2 * padding
+        max-width: 25 * @px_base; // 31 - 2 * padding
+        line-height: 11 * @px_base;
+        z-index: 1;
+    }
+    &[title=""], &:not([title])  {
+        &::after {
+            padding: 0;
+            min-width: 6 * @px_base;
+            min-height: 6 * @px_base;
+            margin: 2 * @px_base;
+            top: 0;
+        }
+    }
+}
+.ui-dropdownmenu-options {
+    li.ui-badge {
+        &::after {
+            top: auto;
+            right: 24 * @px_base;
+            margin-top: 1 * @px_base;
+        }
+    }
+}
+.ui-listview {
+    li {
+        .ui-badge {
+            &::after {
+                top: auto;
+                right: 24 * @px_base;
+                margin-top: 1 * @px_base;
+            }
+        }
+    }
+}
+.ui-main-tab {
+    .ui-badge {
+        display: block;
+        width: auto;
+        position: relative;
+        padding: 0;
+        overflow: initial;
+        height: 17 * @px_base;
+        min-width: 17 * @px_base;
+        margin-left: 3 * @px_base;
+        align-self: center;
+
+        &::after {
+            top: -6 * @px_base;
```

Comment:
Do you process the value in minus to raise the badge a little?

Reply:
This is according to guideline. The badge is positioning 6px above text which indicates.